package.json

```diff
@@ -124,6 +124,7 @@
     "test": "make test",
     "test-client": "NODE_ENV=test NODE_PATH=test:client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client test/runner.js",
+    "test-tests": "NODE_ENV=test NODE_PATH=test:test/test test/runner.js",
```

Comment:
I would have no problem changing this. It doesn't feel inconsistent to me in another sense because the first one we are saying, "test the client" but we only have a single client, same for the server, but the test suite has many. To normalize it my gut reaction would be to use something like `test-test-suite` which I then collapsed into `tests`

But like I said, if you think it's worth changing, I'll be happy to do so.